packages/target-tools/src/utils.js

```diff
@@ -44,6 +46,10 @@ function getNamesForRequested(itemsKey, deliveryRequest) {
       .forEach(item => {
         resultSet.add(item.name);
       });
+
+    if (isPojo(deliveryRequest[type].pageLoad)) {
```

Comment:
`target-global-mbox` should never be added to a list of views, but it would be with this implementation

Reply:
Good catch, thanks! Fixed